Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Build
+FROM rust:1.73-alpine3.18 AS build
+
+WORKDIR /grafbase
+
+COPY ./cli ./cli
+COPY ./engine ./engine
+
+WORKDIR /grafbase/cli
+
+RUN apk add --no-cache git musl-dev
+
+RUN cargo build --release
+
+# Run
+FROM alpine:3.18
+
+WORKDIR /grafbase
+
+RUN apk add --no-cache nodejs npm
+
+COPY --from=build /grafbase/cli/target/release/grafbase /bin/grafbase
+
+ENTRYPOINT ["/bin/grafbase"]
+
+CMD ["start"]
+
```

Comment:
We also should define a volume and use it as workdir. This way its possible to run the container with read only root filesystem while still being able to access and change files inside the volume

Reply:
Can you suggest the change?

Reply:
```
VOLUME [ "/data" ]
WORKDIR "/data"
```